src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,100 @@
-import React from 'react';
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import { getMovie } from '../../api';
 import './FindMovie.scss';
+import { MovieData } from '../../types/MovieData';
+import { MovieCard } from '../MovieCard';
+import { DEFAULT_IMG } from '../../utils';
+import { Movie } from '../../types/Movie';
+
+type Props = {
+  onAddMovieBtn: (newMovie: Movie) => void;
```

Comment:
```suggestion
  onAddMovie: (newMovie: Movie) => void;
```

Reply:
I use 2 function addMovie, and better to show which one is going to perform actions causes directly on button to make it understanble